randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+        ),
+    ],
+    "3162940989": [
+        # Baby Sheegoth Standable Tip
+        # Replaces Baby Sheegoth scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Baby Sheegoth",
+        (
+            "&main-color=#89D6FF;It is possible to stand on Baby Sheegoths."
+            "\n\n\nBy jumping off of them with &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF;, "
+            "one can climb Ice Ruins East without missiles."
+        ),
+    ],
+    "3508298443": [
+        # Bloodflower Standable Tip
+        # Replaces Bloodflower scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Bloodflower",
+        (
+            "&main-color=#89D6FF;Using a double bomb jump, one can roll on top of a Bloodflower.\n\n"
+            "By using another double bomb jump on top of it, "
+            "Great Tree Chamber is reachable from Great Tree Hall without Space Jump Boots."
+        ),
+    ],
+    "2541494929": [
+        # Vault Jump Tip
+        # Replaces Main Plaza fallen stones scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;One can reach the ledge leading to Vault with "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF;.\n\nIt requires holding "
+            "&image=SA,3.0,1.0,1.0,46434ED3,34E79314; while jumping from a brick platform under a twisting tree."
+        ),
+    ],
+    "2903166635": [
+        # Metroid Prime Flamethrower Exo Tip
+        # Replaced Metroid Prime Exo scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Metroid Prime",
+        (
+            "&main-color=#89D6FF;When Metroid Prime's exoskeleton is weak to the Power Beam, "
+            "it can also be damaged by the &main-color=#FF3333;Flamethrower&main-color=#89D6FF;."
+        ),
+    ],
+    "2271263048": [
+        # Hive Mecha Bridge Tip
+        # Replaces Hive Mecha bridge scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        "&main-color=#89D6FF;To cross the bridge without fighting Hive Mecha, carefully walk along its right edge.",
+    ],
+    "2053272936": [
+        # Boost Pipe Speed Tip
+        # Replaces Tallon Canyon halfpipe scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When using the &main-color=#FF3333;Boost Ball&main-color=#89D6FF; in a half pipe, "
+            "hold only &image=8B0C22A7; or &image=778CCD6A; and boost at the lowest point to gain the most height."
+        ),
+    ],
+    "399696251": [
+        # Boost Speed Limit Tip
+        # Replaces Ruined Gallery morph tunnel scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When using the &main-color=#FF3333;Boost Ball&main-color=#89D6FF;, "
+            "if the control stick is kept in a neutral position, the Boost Ball has no speed limit."
```

Comment:
- This scan node seems better suited to explaining the wallboost rather than the mechanics of boost ball.
- Boost Ball's speed cap is also removed if you are holding left or right, not just neutral.

Reply:
That's fair, though I'm not sure where else to put the speed limit explanation. I'd love to find a scan near a long tunnel (like this horseshoe), but that was the only one I could find. Any ideas?